micrometer-core/src/main/java/io/micronaut/configuration/metrics/micrometer/intercept/TimedInterceptor.java

```diff
@@ -199,6 +209,13 @@ private void stopTimed(String metricName, Timer.Sample sample,
             final Timer timer = Timer.builder(metricName)
                     .description(description)
                     .tags(tags)
+                    .tags(
```

Comment:
add a check if the list of taggers is empty and if it is avoid the streaming logic etc.